### PR TITLE
Fixed bottom overflow issue #8728

### LINKED ISSUE #8728: Fix image bottom overflow and const keyword warning in DartPad on 'Implicit animations' codelab
### Page URL

[Page URL](https://docs.flutter.dev/codelabs/implicit-animations)

### Page source

[Page Source](https://github.com/flutter/website/blob/main/src/codelabs/implicit-animations.md)

### Describe the problem

The Fade-in [starter code](https://docs.flutter.dev/codelabs/implicit-animations#fade-in-starter-code) and Fade-in [complete](https://docs.flutter.dev/codelabs/implicit-animations#fade-in-complete) code in iFrames need to be modified to remove the bottom pixels overflow due to the owl image. Moreover const keywords can be added where necessary (The const keyword warning can be seen in the iFrames of both these sections on the page)

Changes needed here: 
### Expected fix

Either of these fixes will work (for both sections): 

1. Wrapping the main column using SingleChildScrollView.

If the requirement is to display the output without any scroll.

2. Adjusting image height to 80% of the screen MediaQuery.

### Additional context

I can file a PR with the changes if permitted. Do let me know the status of the issue. 

Screenshots for reference : 

[Starter Code](https://docs.flutter.dev/codelabs/implicit-animations#fade-in-starter-code)
Screenshot (205)
<br/>
[Complete Code](https://docs.flutter.dev/codelabs/implicit-animations#fade-in-complete)
Screenshot (206)

### PR DESCRIPTION
**Pull request by AKR-2803**
_Description of what this PR is changing or adding, and why:_

This PR is related to image bottom overflow and const keyword warning(#8728). Please review it.

_Issues fixed by this PR (if any):_ 
- Fixes #8728

## Presubmit checklist

- [x] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/main/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

File: src/codelabs/implicit-animations.md [hunk 1]
```diff
@@ -120,8 +121,8 @@ To create a fade-in effect, you can animate the `opacity` property using the
 ```
 
 {{site.alert.info}}
-  You can reference the line numbers in the example code to help track
-  where to make these changes.
+You can reference the line numbers in the example code to help track
+where to make these changes.
```
Comment (sfshaza2) [inline src/codelabs/implicit-animations.md:125]: Actually, we always indent note text by 2 spaces.

```suggestion
  You can reference the line numbers in the example code to help track
  where to make these changes.
```

File: src/codelabs/implicit-animations.md [hunk 2]
```diff
@@ -218,8 +222,8 @@ to set `opacity` to 1:
 ```
 
 {{site.alert.secondary}}
-  Notice that you only need to set the start and end values of `opacity`.
-  The `AnimatedOpacity` widget manages everything in between.
+Notice that you only need to set the start and end values of `opacity`.
+The `AnimatedOpacity` widget manages everything in between.
```
Comment (sfshaza2) [inline src/codelabs/implicit-animations.md:226]: Once again, indent note text by 2 spaces.

```suggestion
  Notice that you only need to set the start and end values of `opacity`.
  The `AnimatedOpacity` widget manages everything in between.
```

**Review by sfshaza2 — CHANGES_REQUESTED** [review]
Hi, @AKR-2803!  Thanks for this PR!  However, I can't accept it, as you have changed too much of our standard formatting. Not only do we indent all "note" text by 2 spaces, but we also. prefer asterisks for lists.

You did appear to run tool/refresh-code-excerpts, so thanks for that!

If you can undo those changes, I can otherwise accept this PR.

**Comment by AKR-2803** [comment]
Hi @sfshaza2, I just noticed it was a code formatting extension in VS code which caused the formatting issues. However, I have comitted necessary changes. Please review it.

**Review by sfshaza2 — APPROVED** [review]
lgtm!

**Comment by sfshaza2** [comment]
@AKR-2803, ack! I see that the build wants you to run tool/refresh-code/excerpts again. Otherwise, this looks great!

**Comment by atsansone** [comment]
@sfshaza2 : I updated two code-excerpts problems that I found locally. I can't see where the others are. Could you point them out?

**Comment by sfshaza2** [comment]
@atsansone, I meant to respond last week. So, I spoke to @parlough about this issue. When you look at the log for the build, it says that 4 examples are a problem, but doesn't identify the exact examples. @parlough, tells me that the build can't really do that. He also stated that fixing these errors manually isn't recommended as they can be quite gnarly. He recommended fixing them using the refresh script. In order to run the refresh script locally, you have to have Docker running and set up. Have you tried that?

**Comment by atsansone** [comment]
Continuing in #9213 as original editor did not return to this PR.

**Pull request closed without merging**